mobile/src/bundles/talent/components/avatar-picker/avatar-picker.tsx

```diff
@@ -33,16 +54,23 @@ const AvatarPicker: React.FC<AvatarPickerProperties> = ({
                     return;
                 }
                 const [image] = assets;
-                setAvatar(image.uri ?? uri);
-            } catch (error) {
-                if (error instanceof Error) {
-                    notifications.showError({ title: error.message });
-                    return;
+                if (image.fileSize && image.type) {
+                    fileSizeValidation(name, image.fileSize);
+                    imageTypeValidation(image.type);
```

Comment:
let's avoid this if else checks in multiple places and just do this check in helpers